### PR TITLE
Fix multiple typos and grammatical errors

### PR DESCRIPTION
**Pull request by SigHunter**
While reading the devmanual, I took some notes when I believe I found mistakes.
Those are - in my opinion - relatively straightforward corrections, mostly grammar and typos. 

I put it all in a single commit, I hope that is OK.

### CODE REVIEW

File: tasks-reference/pam/text.xml [hunk 1]
```diff
@@ -80,14 +80,14 @@ The statement is composed of 3 or 4 tokens:
     methods.  Some documentation states that we need to specify the full path of
     the module, but this creates problems because not all the systems install the
     modules in the same place: Linux-PAM on Gentoo is generally set up to load
-    them from <c>/lib/security</c>, but for example on AMD64 this become
+    them from <c>/lib/security</c>, but for example on AMD64 this becomes
     <c>/lib64/security</c>. The
     result is that providing the full path will lead to non-working <c>pamd</c>
-    files, and the right way to handle this is just states the module name <d /> the
+    files, and the right way to handle this is just to state the module name <d /> the
```
Comment (thesamesam) [inline tasks-reference/pam/text.xml:86]:
```suggestion
    files, and the right way to handle this is to just state the module name <d /> the
```

File: tools-reference/echo/text.xml [hunk 2]
```diff
@@ -29,10 +29,10 @@ the same as the former, but they won't print the trailing newline
 
 <p>
 All usage of the form <c>echo ${somevar} | grep substring</c> just to
-check if the contents of the <c>${somevar}</c> variable
-contains <c>substring</c>, or more often, <c>echo ${somevar} |
+check if the content of the <c>${somevar}</c> variable
```
Comment (thesamesam) [inline tools-reference/echo/text.xml:32]: The changes in this file feel more like style and should perhaps be split into their own commit.

Comment (SigHunter) [reply tools-reference/echo/text.xml:32]: true, I got carried away :-)
I will keep the "third person S correction" in this commit and move the other stuff (commonly, rarely) to what else I'm preparing

File: tasks-reference/pam/text.xml (hunk 1 above)
Comment (SigHunter) [reply tasks-reference/pam/text.xml]: adapted, ty

File: eclass-writing/text.xml [hunk 3]
```diff
@@ -187,7 +187,7 @@ to properly break lines when generating eclass manpages.
 
 <p>
 An eclass is a <c>bash</c> script which is sourced within the ebuild environment.
-Files should be a simple text file with a <c>.eclass</c> extension. Allowed
+Files should be a simple text file with an <c>.eclass</c> extension. Allowed
```
Comment (ulm) [inline eclass-writing/text.xml:190]: This can be pronounced as "a dot eclass extension", in which case "a" is correct.

**Comment by SigHunter** [comment]
first round of edits done from my side, please review, thanks

File: quickstart/text.xml [hunk 4]
```diff
@@ -74,7 +74,7 @@ At the top of the ebuild is a header block. This is present in all ebuilds.
 </p>
 
 <p>
-Ebuilds are indented using tabs, with each tab representing four places.
+Ebuilds are indented using tabs, with each tab representing four spaces.
```
Comment (ulm) [inline quickstart/text.xml:77]: That's not necessarily correct. A tab may represent less than 4 spaces if it doesn't start at a column that is divisible by 4. How about the following wording?
```suggestion
Ebuilds are indented using tabs, with a tab stop every four positions.
```

File: tasks-reference/pam/text.xml [hunk 5]
```diff
@@ -80,14 +80,14 @@ The statement is composed of 3 or 4 tokens:
...
+    them from <c>/lib/security</c>, but for example on AMD64 this becomes
     <c>/lib64/security</c>. The
     result is that providing the full path will lead to non-working <c>pamd</c>
-    files, and the right way to handle this is just states the module name <d /> the
+    files, and the right way to handle this is to just state the module name <d /> the
     PAM implementation will take care of finding the module.
   </li>
   <li>
-    The last token, which can consist of multiple items, describe the parameters
+    The last token, which can consist of multiple items, describes the parameters
```
Comment (ulm) [inline tasks-reference/pam/text.xml:90]: Long line. Please keep them below 80 characters.

File: tasks-reference/pam/text.xml [hunk 6]
```diff
@@ -230,7 +230,7 @@ correct directory to use for the current implementation/arch.
 <p>
 When the PAM module doesn't provide a way to install the package by itself, such
 as a <c>Makefile</c> or an installation script, there are also the <c>dopammod</c> and
-<c>newpammod</c> functions which takes care of install the module in the right
+<c>newpammod</c> functions, which take care of installing the module in the right
```
Comment (ulm) [inline tasks-reference/pam/text.xml:233]: Long line.

**Review by ulm — CHANGES_REQUESTED** [review]

File: quickstart/text.xml (hunk 4 above)
Comment (SigHunter) [reply quickstart/text.xml:77]: I was using [ebuild-writing/file-format](https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#indenting-and-whitespace) as reference, where it says spaces

Comment (ulm) [reply quickstart/text.xml:77]: Hm, I guess for indenting it is correct.

File: eclass-writing/text.xml (hunk 3 above)
Comment (SigHunter) [reply eclass-writing/text.xml]: I see. reverted

File: tasks-reference/pam/text.xml (hunk 5 above)
Comment (SigHunter) [reply tasks-reference/pam/text.xml]: should fit now, thanks for spotting

**Review by ulm — APPROVED** [review]

**Comment by ulm** [comment]
Thank you, good to go from my side.
If there are no further comments from @thesamesam then I'm going to merge this.

**Review by thesamesam — APPROVED** [review]

**Comment by ulm** [comment]
Merged in commit c08a1dc1072740182e5c4d059967db697897bed6.
Thanks again.
